notebooks/04_filter.ipynb

```diff
@@ -0,0 +1,1008 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    compute_image_statistics(dataset)
</pre><p>After merging the <strong>repr</strong> first, could you replace this by printing dataset instead of using compute_image_statistics?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>